### ISSUE TITLE
Login Page - Accessibility 

### ISSUE THREAD

**Issue by access42** [YOURLS/YOURLS#3653]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Self troubleshooting

- [X] I've read the [documentation](https://yourls.org/) and made sure to follow it.
- [X] I've read the [troubleshooting first steps](https://docs.yourls.org/guide/troubleshooting/first-steps.html) and [frequent issues](https://docs.yourls.org/guide/troubleshooting/frequent-issues.html) guides.

### Version

1.9.2

### Description

The login page has accessibility issues according to the WCAG.

- [ ] #3646
- [ ] #3647
- [ ] #3648
- [ ] #3649
- [ ] #3650
- [ ] #3651 
- [ ] According to the [WCAG 2.4.2 criteria](https://www.w3.org/TR/WCAG21/#page-titled), web pages have titles that describe topic or purpose.

The page has a title but does not allow users to know its purpose (here, the login page)

<img width="454" alt="" src="https://github.com/YOURLS/YOURLS/assets/8665093/b3fcbcdb-ac86-4e82-b928-46ecf57da933">

To allow users to know what page it is easily, the title of the page could be changed to “Login – YOURLS – Your Own URL Shortener”.

- [ ] According to the [WCAG 3.3.1 criteria](https://www.w3.org/TR/WCAG21/#error-identification), if an input error is automatically detected, the item that is in error is identified and the error is described to the user in text.

<img width="163" alt="image" src="https://github.com/YOURLS/YOURLS/assets/8665093/20dbb49b-4b25-4b3c-a34c-634ea96dfdfd">

The form should have visible text instructions at the beginning of the form that describes the necessary inputs.

The error message should be linked to the input using `aria-describedby` to allow assistive technologies to vocalize the error to the user.
